content/en/docs/contribute/new-content/new-features.md

```diff
@@ -0,0 +1,139 @@
+
+### Feature tracking sheet
+
+The feature tracking sheet
+[for a given Kubernetes release](https://github.com/kubernetes/sig-release/tree/master/releases) lists each feature that is planned for a release.
+Each line item includes the name of the feature, a link to the feature's main
+GitHub issue, its stability level (Alpha, Beta, or Stable), the SIG and
+individual responsible for implementing it, whether it
+needs docs, a draft release note for the feature, and whether it has been
+merged. Keep the following in mind:
+
+- Beta and Stable features are generally a higher documentation priority than
+  Alpha features.
+- It's hard to test (and therefore, document) a feature that hasn't been merged,
+  or is at least considered feature-complete in its PR.
+- Determining whether a feature needs documentation is a manual process and
+  just because a feature is not marked as needing docs doesn't mean it doesn't
+  need them.
+
+## For developers or other SIG members
+
+This section is information for members of other Kubernetes SIGs documenting new features
+for a release.
+
+If you are a member of a SIG developing a new feature for Kubernetes, you need
+to work with SIG Docs to be sure your feature is documented in time for the
+release. Check the
+[feature tracking spreadsheet](https://github.com/kubernetes/sig-release/tree/master/releases)
+or check in the `#sig-release` Kubernetes Slack channel to verify scheduling details and
+deadlines.
+
+### Open a placeholder PR
+
+1. Open a pull request against the
+`release-X.Y` branch in the `kubernetes/website` repository, with a small
+commit that you will amend later.
+2. Use the Prow command `/milestone X.Y` to
+assign the PR to the relevant milestone. This alerts the docs person managing
+this release that the feature docs are coming.
+
+If your feature does not need
+any documentation changes, make sure the sig-release team knows this, by
+mentioning it in the `#sig-release` Slack channel. If the feature does need
+documentation but the PR is not created, the feature may be removed from the
+milestone.
+
+### PR ready for review
+
+When ready, populate your placeholder PR with feature documentation.
+
+Don't worry about polish or formatting, just describe what the
+feature does and how to use it. The docs
+person managing the release will work with you to get the content into shape
+to be published.
+
+If your feature needs documentation and the first draft
+content is not received, the feature may be removed from the milestone.
+
+### All PRs reviewed and ready to merge
+
+If your PR has not
+yet been merged into the `release-X.Y` branch by this deadline, work with the
+docs person managing the release to get it in. If your feature needs
+documentation and the docs are not ready, the feature may be removed from the
+milestone.
+
+If your feature is an Alpha feature and is behind a feature gate, make sure you
+add it to [Feature gates](/docs/reference/command-line-tools-reference/feature-gates/)
```

Comment:
nit: feature gates is a page that contains tables

Reply:
I'm not sure I understand the nit. Are we trying to move away from using tables?

Reply:
I'll have to reread. The sentence seemed incomplete for some reason.
No, not moving away from tables. There is a Feature gates page that has a number of tables (one table for alpha and beta feature gates, another table for features that are deprecated or have reached GA).